src/core/utils/parse_cmdline.cpp

```diff
@@ -33,12 +33,73 @@
 
 #include "parse_cmdline.hpp"
 
-#include <limits>
 #include <string.h>
 
 #include "common/code_utils.hpp"
 #include "net/ip6_address.hpp"
 
+namespace {
```

Comment:
How about adding this as a new common header e.g., `common/numeric_limits.hpp`?
This way it can also be used by other core modules.

Reply:
Done.